### ISSUE TITLE
Missing keys while loading pre-trained FCOS-3D weights

### ISSUE THREAD

**Issue by amundra15** [wzzheng/TPVFormer#58]
While loading the pre-trained FCOS-3D weights for training, the code complains about incompatible keys.

Upon analyzing it, most of the "incompatible"  keys are as expected (eg, `tpv_aggregator`, `tpv_head` related keys missing in the loaded model and `bbox_head` keys missing in the defined model). 

However, one set of items still did not make sense to me. For the `image_backbone` (and possibly for `image_neck`) batch norm layers, the `running_mean`, `running_var`, and `num_batches_tracked` keys are present in the checkpoint, but not in my model definition. I am not sure if this is intended -- a quick Google search says that these parameters are critical for training and inference, even if the BN parameters are frozen.

Could you shed some light on this?
